systems/registry/node/pkg/config.go

```diff
@@ -62,6 +62,7 @@ func NewConfig(name string) *Config {
 		},
 		Service: uconf.LoadServiceHostConfig(name),
 		MsgClient: &uconf.MsgClient{
+			// Host:    "msgclient-registry:9095",
```

Comment:
This msg client host comment can be removed.

Reply:
addressed